pkg/install/app/app.go

```diff
@@ -0,0 +1,24 @@
+package app
+
+import (
+	"github.com/puppetlabs/leg/k8sutil/pkg/controller/ownerext"
+	"github.com/puppetlabs/leg/k8sutil/pkg/norm"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	webhookTLSDirPath     = "/var/run/secrets/puppet/relay/webhook-tls"
+	jwtSigningKeyDirPath  = "/var/run/secrets/puppet/relay/jwt"
+	jwtSigningKeyPath     = "/var/run/secrets/puppet/relay/jwt/private-key.pem"
+	metadataAPITLSDirPath = "/var/run/secrets/puppet/relay/tls"
+)
+
+var DependencyManager = ownerext.NewManager("installer.relay.sh/dependency-of")
+
+// TODO share this with pkg/operator/app
```

Comment:
=)

Reply:
Just an FYI, this is done in the next branch. I moved the function to leg/k8sutils.

Reply:
Yeah, I know. Just adding a reminder, just in case.